sound_manager.py

```diff
@@ -0,0 +1,64 @@
+from arcade import Sound, load_sound
+
+DEFAULT_VOLUME = 0.7
+SFXPATH_BEETLE_HIT = "Assets/Sound/SFX/impact2_noisecollector.wav"
+SFXPATH_BEETLE_DEAD = "Assets/Sound/SFX/beetle_dead_mixkit.wav"
+SFX_PATH_PEASHOOTER = "Assets\Sound\SFX\pew_xtrgamr.wav"
+
+class SoundManager:
+    def __init__(self):
+        self.audio_player = Sound()
+        self.sfx_beetle_dead = load_sound(SFXPATH_BEETLE_DEAD)
+        self.sfx_beetle_hit = load_sound(SFXPATH_BEETLE_HIT)
+        self.peashooter_pew = load_sound(SFX_PATH_PEASHOOTER)
+
+    def setup(self):
+        self.set_volume(DEFAULT_VOLUME)
+
+    def set_volume(self, volume=None):
+        if not volume:
+            self.audio_player.set_volume = 0.7
+        else:
+            self.audio_player.set_volume = volume
+
+    def play_sound(self, sound):
+        if not sound:
+            pass
+        else:
+            self.audio_player.play(sound)
```

Comment:
From reading the docs, I think this and all the other places like this should be `arcade.play_sound`. Right now, I think you're creating a "blank" sound, and then trying to pass in another sound, but [looking at the docs](https://api.arcade.academy/en/latest/api/sound.html#arcade.Sound.play), I don't think that's going to work.

I'll write up more about my idea in the main comment.

Reply:
I had problems with arcade.play_sound when I tried to use any of the other arcade.Sound methods on it--it doesn't return a player object, but Sound.audio_player.play does. I may have to look into the docs a bit more to grasp how to use audio_player.play, but I am certain that's the right one.

Reply:
Rereading the doc, it looks like play_sound does create the player? I'll have to test with this some more.

Reply:
Huh, it's possible I'm wrong and you need to use that...

Reply:
Either way, I trust whatever works here, so I'm going to resolve this conversation.